dist/oauth2-redirect.html

```diff
@@ -33,7 +33,41 @@
             if (qp.code) {
                 delete oauth2.state;
                 oauth2.auth.code = qp.code;
-                createForm(oauth2.auth, qp).submit();
+                var form = new FormData(createForm(oauth2.auth, qp));
+                var headers = new Headers();
+
+                headers.append("Access-Control-Allow-Origin", "*");
+                fetch(oauth2.auth.schema.get("tokenUrl"), {
+                  method: "POST",
+                  headers: headers,
+                  mode: "cors",
+                  body: form
+                }).then(function (res) {
+                    var status = res.status
+                    if (status >= 300 && status < 400) {
+                      window.location = res.headers && res.headers.location
```

Comment:
I think `res.headers` is guaranteed to exist.
Is this _native fetch_ or swagger-js's `fetch`?